src/commands.jl

```diff
@@ -51,6 +51,7 @@ function execute_command(state::DebuggerState, ::Union{Val{:c},Val{:nc},Val{:n},
                 state.broke_on_error = true
             end
         end
+        JuliaInterpreter.maybe_next_call!(state.frame)
```

Comment:
I think this is a good idea. Have you checked what happens, if there isn't another call? E.g., you put a `@bp` at function exit?

Reply:
Yeah, this will just advance to the `return` statement.

```jl
> function f(x)
      x = sin(2.0)
      @bp
  end
f (generic function with 1 method)

> @run f(2)
Hit breakpoint: f(x) in Main at REPL[3]:2, line 3
In f(x) at REPL[3]:2
2     x = sin(2.0)
3     @bp
4 end

About to run: return
1|debug> 

> function f(x)
      x = sin(2.0)
      return @bp
  end
> f (generic function with 1 method)

> @run f(2)
Hit breakpoint: f(x) in Main at REPL[5]:2, line 3
In f(x) at REPL[5]:2
2     x = sin(2.0)
3     return @bp
4 end

About to run: return
1|debug> n

>
```